AK/JsonParser.cpp

```diff
@@ -48,32 +48,48 @@ String JsonParser::consume_and_unescape_string()
         if (next_is('\0')) {
             return {};
         }
+        
+        if (next_is('n')) {
             final_sb.append('\n');
-            break;
-        case 'r':
+            continue;
+        }
+        if (next_is('r')) {
             final_sb.append('\r');
-            break;
-        case 't':
+            continue;
+        }
+        if (next_is('t')) {
             final_sb.append('\t');
-            break;
-        case 'b':
+            continue;
+        }
+        if (next_is('b')) {
             final_sb.append('\b');
-            break;
-        case 'f':
+            continue;
+        }
+        if (next_is('f')) {
             final_sb.append('\f');
-            break;
-        case 'u': {
+            continue;
+        }
+        if (next_is('u')) {
+            if (tell_remaining() < 4) {
+                return {};
+            }
```

Comment:
Please omit the curly braces if they aren't needed (except if it's for symmetry or there are nested `if`/`for`/... statements.
```suggestion
            if (tell_remaining() < 4)
                return {};
```